generated/php/google-cloud-error-reporting-v1beta1/proto/src/Google/Devtools/Clouderrorreporting/V1beta1/DeleteEventsRequest.php

```diff
@@ -11,7 +11,7 @@
 /**
  * Deletes all events in the project.
  *
- * Protobuf type <code>Google\Devtools\Clouderrorreporting\V1beta1\DeleteEventsRequest</code>
+ * Generated from protobuf message <code>google.devtools.clouderrorreporting.v1beta1.DeleteEventsRequest</code>
```

Comment:
Yes, but this is referring to the protobuf message type, which is lowercase and `.` separated. This is a change in PHP protoc which we are now picking up in our generated proto classes.